src/hooks/UseIsLoggedIn.jsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+import { Outlet, useNavigate } from 'react-router-dom';
+
+import { getCookie } from '../hooks/useCookie';
+
+const UseIsLoggedIn = () => {
+    const navigate = useNavigate();
+
+    const loggedInUser = getCookie('authToken');
+
+    useEffect(() => {
+        if (loggedInUser) {
```

Comment:
Couldn't we reutilize this one for redirecting to home or login depending on the result of loggedInUser?

Reply:
done